components/collapse/style/index.less

```diff
@@ -3,9 +3,6 @@
 
 @collapse-prefix-cls: ~"@{ant-prefix}-collapse";
 
-@collapse-header-bg: @background-color-light;
```

Comment:
You shouldn't change the original variable name cause someone may use it already.

Reply:
Good catch. Fixed a few lines below.